egs2/iwslt22_low_resource/st1/RESULTS.md

```diff
@@ -0,0 +1,26 @@
+<!-- Generated by scripts/utils/show_translation_result.sh -->
+# RESULTS
+## Environments
+- date: `Fri Mar 10 16:10:20 CST 2023`
+- python version: `3.8.16 (default, Jan 17 2023, 23:13:24)  [GCC 11.2.0]`
+- espnet version: `espnet 202301`
+- pytorch version: `pytorch 1.13.1`
+- Git hash: `ff841366229d539eb74d23ac999cae7c0cc62cad`
+  - Commit date: `Mon Feb 20 12:23:15 2023 -0500`
+
+## st_wav2vec-transformer-warmup-15k
+
+### BLEU
+
+|dataset|score|verbose_score|
+|---|---|---|
+|decode_pen2_st_model_valid.acc.ave/test|2.6|22.5/4.5/1.8/0.8 (BP = 0.736 ratio = 0.765 hyp_len = 17223 ref_len = 22504)|
```

Comment:
The ratio is too different (hypotheses are too short) to me.
Can you tune the length penalty (in later PR?)?

Reply:
Of course. No problem.